packages/terra-infinite-list/src/terra-dev-site/test/infinite-list/InfiniteListSameCount.test.scss

```diff
@@ -0,0 +1,23 @@
+@import './InfiniteListTestCommon';
+
+:local {
+  .item-type1 {
+    @include content-common(50px, #ffe4c4);
+  }
+
+  .item-type2 {
+    @include content-common(50px, #b8860b);
+  }
+
+  .item-type3 {
+    @include content-common(50px, #5f9ea0);
+  }
+
+  .item-type4 {
+    @include content-common(50px, #dc143c);
```

Comment:
lets drop the @include syntax and directly add the styles here

Reply:
Done